src/Tooling/Bit.Tooling.Templates/TodoTemplate/App/TodoTemplate.App.csproj

```diff
@@ -32,15 +32,16 @@
     <PropertyGroup Condition="$(TargetFramework.Contains('-android')) and '$(Configuration)' == 'Release'">
         <EnableLLVM>true</EnableLLVM>
         <AndroidPackageFormat>apk</AndroidPackageFormat>
-        <RuntimeIdentifiers>android-arm;android-arm64</RuntimeIdentifiers>
-        <MauiUseDefaultAotProfile>false</MauiUseDefaultAotProfile>
+        <RuntimeIdentifiers>android-arm</RuntimeIdentifiers>
+        <MauiUseDefaultAotProfile Condition="Exists('custom.aprof')">false</MauiUseDefaultAotProfile>
```

Comment:
again, this change is not related to this issue.
plz remove it and submit a new issue.